src/operator/elemwise_op_common.h

```diff
@@ -156,6 +157,61 @@ inline bool ElemwiseStorageType(const nnvm::NodeAttrs& attrs,
     attrs, in_attrs, out_attrs);
 }
 
+template<int n_out>
```

Comment:
1) validation that expected number of outputs is the actual number of outputs being processed
2) this is consistent with the other 5-6 storage type inference functions surrounding this function